src/main/java/pro/quicksense/modules/service/ISysUserService.java

```diff
@@ -0,0 +1,12 @@
+package pro.quicksense.modules.service;
+
+import com.baomidou.mybatisplus.extension.service.IService;
+import pro.quicksense.modules.common.Result;
+import pro.quicksense.modules.entity.User;
+
+
+public interface ISysUserService extends IService<User> {
```

Comment:
```suggestion
public interface UserService extends IService<User> {
```
also modify the file name.